ROADMAP.md

```diff
@@ -6,53 +6,59 @@ As a public community project the KitOps roadmap is always open to feedback. If
 * Tell us in the [KitOps Discord](https://discord.gg/Tapeh8agYy) channel
 * Send us a message on [X / Twitter](https://twitter.com/Kit_Ops)
 
-## Roadmap
-🛳️ = Shipped
+## In-Progress
 
-🚗 = In-progress
+* Create a [Homebrew cask for KitOps](https://github.com/jozu-ai/homebrew-kitops/issues/6)
+* Create [Python SDK for KitOps](https://github.com/jozu-ai/kitops/issues/118)
+* [Improve handling of TAR files](https://github.com/jozu-ai/kitops/issues/489) generated by Kit
+* Demo: using [ModelKits with MLFlow](https://github.com/jozu-ai/kitops/issues/545)
+* Convert a model from [Hugging Face to ModelKit](https://github.com/jozu-ai/kitops/issues/564)
 
-📅 = Planned
 
-💡 = Idea (please provide feedback)
+## Recently Completed
+See the full changelog from [0.4](https://github.com/jozu-ai/kitops/releases/tag/v0.4.0) and [0.34](https://github.com/jozu-ai/kitops/releases/tag/v0.3.4)
 
-### Dev Mode Improvements
+* Demo: using [ModelKits with Jupyter Notebooks](https://youtu.be/OQPp7QEvk7Q?feature=shared)
+* Add [Windows & Linux for `kit dev`](https://github.com/jozu-ai/kitops/releases/tag/v0.4.0) command
+* [Add `--filter` option](https://github.com/jozu-ai/kitops/releases/tag/v0.4.0) to `kit info` command
+* Deploy [ModelKits as containers](https://kitops.ml/docs/deploy.html)
+* [Added parameters field](https://github.com/jozu-ai/kitops/pull/514) to datasets and code
+* [Enhanced proxy](https://github.com/jozu-ai/kitops/pull/512) support
 
-* Change `kit dev` implementation to use [Llamafile](https://github.com/Mozilla-Ocho/llamafile?tab=readme-ov-file) 📅
-* Export [Llamafile](https://github.com/Mozilla-Ocho/llamafile?tab=readme-ov-file) from ModelKit 📅
-* Generate example code as the parameters and prompts are entered 💡
-* A way to see JSON communicated between the server and responses 💡
-* Hide the parameters that are not frequently changed 💡
-* Add `kit dev` command to Kit CLI on *Windows* 💡
-* Add `kit dev` command to Kit CLI on *Linux* 💡
+## Roadmap Themes
 
-### Tutorials
+### UX Improvements
 
-#### Demos
+* [Kit CLI should be able to reuse Docker credentials](https://github.com/jozu-ai/kitops/issues/213)
+* Export [Llamafile](https://github.com/Mozilla-Ocho/llamafile?tab=readme-ov-file) from ModelKit
+* Kit Dev: Generate example code as the parameters and prompts are entered
+* Kit Dev: A way to see JSON communicated between the server and responses
+* Kit Dev: Hide parameters that are not frequently changed
 
```

Comment:
I think we should add the pack from HF in here.

Reply:
🤦 cant' beleive I missed my favourite feature request!